code/game/objects/structures/window_frames.dm

```diff
@@ -54,17 +54,47 @@
 /obj/structure/window_frame/Initialize(mapload)
 	. = ..()
 
+	if(grille)
+		grille = new grille(src, sheet_amount)
```

Comment:
you'd probably want to istype this just for clarity

Reply:
What?